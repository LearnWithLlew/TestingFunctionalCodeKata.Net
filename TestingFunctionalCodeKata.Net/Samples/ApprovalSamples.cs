using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using ApprovalTests;
using ApprovalTests.Reporters;
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace TestingFunctionalCodeKata.Net.Samples
{
    [TestClass]
    [UseReporter(typeof(DiffReporter))]
    public class ApprovalSamples
    {
        [TestMethod]
        public void TestVerifyAll()
        {
            Approvals.VerifyAll("sin", new[] { 0.1 }, d => $"sin({d}) = {TrigMath.Sin(d)} ");
        }
    }
}
