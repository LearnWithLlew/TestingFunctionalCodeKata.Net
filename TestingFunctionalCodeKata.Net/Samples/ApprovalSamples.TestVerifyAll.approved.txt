sin

sin(0.1) = 0.09983278846736755 
